### PR TITLE
CRUCIAL workplace safety equipment: Recolorable overalls

### PR DESCRIPTION
**Pull request by Paxilmaniac**
## About The Pull Request

It's not really that clamplicated, quite shrimple actually, it just adds recolorable overalls as a suit item to the loadout.

## How This Contributes To The Skyrat Roleplay Experience

We have a DISTINCT lack of bitches in overalls, I need this fixed NOW.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

image
  
</details>

## Changelog

:cl:
add: A set of greyscale overalls has been added to the loadout menu in the suits section
/:cl:

### CODE REVIEW

**Review by Ebin-Halcyon — CHANGES_REQUESTED** [review]
Making them an oversuit layers them over hands and boots.
Make some cutouts for the hands, adjust the layer priority to have boots layer over them if you want that. Otherwise you can just make them a uniform and disregard everything above.

**Comment by PCPort** [comment]
I personally support the "make it a uniform" idea because being able to wear jackets over the overalls would be funny and possibly cute.

**Review by Ebin-Halcyon — APPROVED** [review]

File: modular_skyrat/modules/customization/modules/clothing/suits/misc.dm [hunk 1]
```diff
@@ -320,3 +320,40 @@
...
+/obj/item/clothing/suit/apron/overalls/greyscale/AltClick(mob/user)
+	. = ..()
+	if(!iscarbon(user))
+		return
+	var/mob/living/carbon/carbon_user = user
+	if(carbon_user.get_item_by_slot(slot_flags) == src)
+		to_chat(user, span_warning("You must take [src] off before adjusting it!"))
+		return
+	if(!user.is_holding(src))
+		to_chat(user, span_warning("You must be holding [src] in order to adjust it!"))
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/clothing/suits/misc.dm:349]: Why?

Comment (Paxilmaniac) [reply modular_skyrat/modules/customization/modules/clothing/suits/misc.dm:349]: source: its what bandanas do

**Review by tf-4 — APPROVED** [review]

**Pull request merged**